src/test/resources/reporting/generatedoutput-testing-feature.ttl

```diff
@@ -0,0 +1,46 @@
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix td: <http://www.w3.org/2006/03/test-description#>
+prefix earl: <http://www.w3.org/ns/earl#>
+prefix dcterms: <http://purl.org/dc/terms/>
+prefix spec: <http://www.w3.org/ns/spec#>
+prefix prov: <http://www.w3.org/ns/prov#>
+prefix ex: <https://example.org/>
+
+ex:scenario1-output
+    a prov:Entity ;
+    prov:generatedAtTime "2021-04-06T17:41:20.889+01:00"^^xsd:dateTime ;
```

Comment:
Let's stick to "Z" with microsecond precision.

Reply:
OK